examples/postprocessing/postprocessor_validate.yaml

```diff
@@ -0,0 +1,66 @@
+accelerator: gpu
+precision: 32
+devices: 1
+
+dataset_root: /path/to/dataset  # <---- specify path to your dataset
+dataframe_name: df.csv
+
+bs_val: 128
+num_workers: 20
+
+extractor_weights: /path/to/extractor.ckpt  # <---- specify path to a baseline feature extractor
+postprocessor_weights: /path/to/postprocessor.ckpt  # <---- specify path to a trained postprocessor
+
+transforms_val:
+  name: norm_resize_hypvit_torch
+  args:
+    im_size: 224
+    crop_size: 224
+
+model:
+  name: vit
+  args:
+    arch: vits16
+    normalise_features: True
+    use_multi_scale: False
+    weights: ${extractor_weights}
+
+postprocessor:
+  name: pairwise_images
+  args:
+    top_n: 5
+    pairwise_model:
+      name: concat_siamese
+      args:
+        mlp_hidden_dims: [192]
+        weights: ${postprocessor_weights}
+        extractor:
+          name: vit
+          args:
+            arch: vits16
+            normalise_features: False
```

Comment:
false, true?

Reply:
False is correct, because it's inside the pairwise model